README.md

```diff
@@ -590,6 +590,7 @@ Instead, you can ignore notices based on some condition.
 ```ruby
 Airbrake.add_filter do |notice|
   notice.ignore! if notice.stash[:exception].is_a?(StandardError)
+  notice.ignore! if notice.stash[:exception].message.match(/Couldn't find ActiveStorage::Blob with 'id'/)
```

Comment:
No objections to the change but I would prefer to split this into multiple lines since now this line too long (rubocop would complain).

Reply:
good point, I'll refactor this after we settle discussion https://github.com/airbrake/airbrake-ruby/pull/649#discussion_r640751906